meta-facebook/minerva-ag/src/shell/plat_thermaltrip_switch_shell.c

```diff
@@ -45,45 +45,71 @@ void cmd_thermaltrip_status_get(const struct shell *shell, size_t argc, char **a
 	shell_print(shell, "thermaltrip switch %s", (msg.data[0]) ? "enable" : "disable");
 }
 
-static uint8_t switch_set(const struct shell *shell, bool is_enable)
-{
-	I2C_MSG msg = { 0 };
+// static uint8_t plat_thermaltrip_set(const struct shell *shell, bool is_enable, bool is_perm)
```

Comment:
@LisaChang-Quanta instead of commenting, can we just delete this code if we are not using it anymore